File: Exavision.Seasense.Shared/Settings/SettingDoubleValueCapability.cs
using Exavision.Seasense.Shared.Capabilities;

namespace Exavision.Seasense.Shared.Settings {
    public class SettingDoubleValueCapability : SettingCapability {

        public double MinValue { get; set; }

        public double MaxValue { get; set; }

        public DoubleValueType DoubleValueType { get; set; }
    }
}

File: Exavision.Seasense/Client/src/app/materials/settings/setting-double-value-capability.ts
import { CapabilityType } from "../capabilities/capability-type";
import { DoubleValueType } from "../capabilities/double-value-type";
import { SettingCapability } from "./setting-capability";

export interface SettingDoubleValueCapability extends SettingCapability {
  id: string;
  capabilityType: CapabilityType;
  doubleValueType: DoubleValueType;
  minValue: number;
  maxValue: number;
  displayName: string;
  type: string;

}


